frontend/src/App.tsx

```diff
@@ -1,21 +1,26 @@
-import { FormEvent, useCallback, useEffect, useState } from "react";
+import { FormEvent, useCallback, useRef, useEffect, useState } from "react";
 
 import "./App.css";
 import { Todo } from "./models";
 import { deleteTodo, changeTodo, addTodo, fetchTodos } from "./apis/";
-import { Filters, Input, TodoList, Category } from "./components";
+import { Filters, Input, TodoList, Category, Alert } from "./components";
 
 function App() {
   const [todos, setTodos] = useState<Todo[]>([]);
   const [newTodoDescription, setNewTodoDescription] = useState("");
   const [loading, setLoading] = useState(true);
   const [selectedFilter, setSelectedFilter] = useState<Category>(Category.ALL);
+  const inputRef = useRef<HTMLInputElement | null>(null);
+  const [error, setError] = useState<boolean>(false);
+
+  document.title = "Todo App";
```

Comment:
This line will run **each** time the `App` component re-renders, which means the title will be set during **each** render cycle.

Make a modification here that will ensure that this line only runs **once**.